sydent/pipeline.yml

```diff
@@ -1,18 +1,24 @@
 steps:
   - command:
-    - "/sydent-ci.sh"
-    label: "Check Sydent starts"
+      - "echo '--- Install dependencies'"
```

Comment:
might be better to update `sydent-ci.sh` instead?

Reply:
Thing is I don't actually know where that script is...

Reply:
The script exists in the closed PR that also brought about this docker image. https://github.com/matrix-org/sydent/pull/193/files#diff-c2bd4c3d264a03f6f7582437f460d552

So it never became part of the repo.

Reply:
fair enough.

I still think it would be a good idea for these lists of commands to be in the `sydent` git repo; or (in this case) maybe make `tox` work and this can then be

>      - "python -m pip install tox"
>      - "tox -e py27"
 
or something.

still, I guess we're going to be iterating here so this is an improvement for now.